examples/apps/data-object-grid/src/icons.tsx

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	CodeTextRegular,
+	CursorClickRegular,
+	DocumentTextRegular,
+	DrawTextRegular,
+	NumberSymbolRegular,
+} from "@fluentui/react-icons";
+import React from "react";
+
+// Map icons names to their instantiated React element
+export const iconMap = {
+	Code: <CodeTextRegular />,
+	Edit: <DocumentTextRegular />,
+	FabricTextHighlight: <DrawTextRegular />,
+	NumberSymbol: <NumberSymbolRegular />,
+	Touch: <CursorClickRegular />,
+};
```

Comment:
Previously, the FluentUI icons were referenced by string.  Now they are encapsulated in React components.

The sample seems to be making an effort to separate model and UI, so rather than mix the React components into the app's toolbar registry, I added a map from "icon name" -> "React component".